marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -324,12 +324,26 @@ function convertStepReferenceToInlineStep(stepId) {
   return newFlowStep;
 }
 
+function getAllArtifactCollections() {
```

Comment:
If we keep this feature, let's not add a function to this library when we only have one use for it.